runtime/themes/dark_plus.toml

```diff
@@ -79,6 +79,7 @@
 "ui.text.focus" = { fg = "white" }
 
 "ui.virtual" = { fg = "dark_green" }
```

Comment:
Do you have opinions on the new `ui.virtual` scope here? Trying out this change, I think

```toml
"ui.virtual" = { fg = "borders" }
```

Might be a better highlight. Wdyt? (`:set whitespace.render all` to turn on the whitespace rendering that uses it)

Reply:
Ah that looks great! Would you mind adding that change here?

Reply:
Fixed!